tools/appscript/add-custom-menu.mjs

```diff
@@ -18,18 +18,26 @@ export default function () {
     .addSubMenu(
       SpreadsheetApp.getUi()
         .createMenu('Advanced')
-        .addItem('Create GitHub repository', 'createRepository')
         .addItem('Revalidate metadata / adopted schedule (after manual edits)', 'validateGrid')
-        .addItem('Recover published schedule (from GitHub)', 'importGrid')
-        .addItem('Fetch event info, rooms, days, slots (from GitHub)', 'importMetadata')
-        .addItem('Export event info, rooms, days, slots (to GitHub)', 'exportMetadata')
-        .addItem('Export event to files (HTML, JSON)', 'exportEventToFiles')
+        .addItem('Refresh list of registrants (Groups only)', 'fetchRegistrants')
```

Comment:
I note we should hardly ever need that menu from now on. It's convenient for debugging, but refreshing sessions/groups (main menu) also refreshes the list of registrants.